scripts/kafka-integration-test.sh

```diff
@@ -5,36 +5,53 @@
 
 set -euf -o pipefail
 
-compose_file="docker-compose/kafka-integration-test/docker-compose.yml"
+compose_file=""
 jaeger_version=""
+kafka_version=""
 manage_kafka="true"
 success="false"
 
-print_help() {
-  echo "Usage: $0 [-K] -j <jaeger_version>"
+usage() {
+  echo "Usage: $0 [-K] -j <jaeger_version> -v <kafka_version>"
   echo "  -K: do not start or stop Kafka container (useful for local testing)"
   echo "  -j: major version of Jaeger to test (v1|v2)"
+  echo "  -v: kafka major version (3.9|2.8)"
```

Comment:
Specify the exact expected values. The exact version like 3.9 is controlled by the docker-compose file, not by this script.
```suggestion
  echo "  -v: kafka major version (3.x|2.x)"
```